apps/server/src/service/dexLiquidityPoolService.ts

```diff
@@ -288,6 +311,7 @@ export class DexLiquidityPoolService {
       tokenB,
       infoCell,
       new Token(new Script(SUDT_TYPE_CODE_HASH, SUDT_TYPE_HASH_TYPE, infoCell.cellOutput.lock.toHash()).toHash()),
+      infoCell.blockNumber === '0' ? 'pending' : 'success',
```

Comment:
The status of ckb's transactions includes `pending`, `proposed` and `completed`, so perhaps we should use a uniform `status` as much as possible to avoid ambiguity

Reply:
ok

Reply:
> The status of ckb's transactions includes `pending`, `proposed` and `completed`, so perhaps we should use a uniform `status` as much as possible to avoid ambiguity

ok